README.md

```diff
@@ -87,6 +87,13 @@ assertDrawerIsClosed(R.id.drawer);
 // And another tricky feature
 assertThatBackButtonClosesTheApp();
 ```
+
+# Download
+
+```groovy
```

Comment:
Sorry for my confused @rocboronat, It should be 'gradle'. 
I copied 'groovy' from other projects and only change artifact id. :(